src/main/java/Nateh.java

```diff
@@ -0,0 +1,107 @@
+import java.util.Scanner;
+
+public class Nateh {
+    public static String lineBreak = "____________________________________________________________\n";
```

Comment:
Perhaps lineBreak could be a constant, along with the appriopriate naming convention? Otherwise, I like the idea of having such a constant for reusability!